OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2249,6 +2249,14 @@ internal static void UpgradeWeaponRules(int engineVersion, ref List<MiniYamlNode
 					}
 				}
 
+				if (engineVersion < 20150828)
+				{
+					if (depth == 2 && parentKey == "Projectile" && !node.Key.StartsWith("Sequences") && node.Key.StartsWith("Sequence"))
```

Comment:
Ah, didn't realize there were projectiles other than `Bullet` and `Shrapnel`. Glad you caught that.